### ISSUE TITLE
v1.43.0 Regression: `EmptyExampleGroup` incorrectly flags custom example groups w/ block arguments

### ISSUE THREAD

**Issue by jtannas** [rubocop-hq/rubocop-rspec#1006]
Running v1.43.1, I've run into a regression where the `EmptyExampleGroup` cop is now incorrectly flagging custom includes that take a block argument. In my case, it's a `run_test!` method that makes a request and expects the response to match a swagger spec, and optionally takes additional expectations within a block.

Reproducible with the following failing test:
```ruby

  context 'when a custom include method takes a block argument' do
    let(:cop_config) do
      { 'CustomIncludeMethods' => ['run_test!'] }
    end

    it 'ignores an empty example group with a custom include and a block' do
      expect_no_offenses(<<~RUBY)
        describe Foo do
          context "when I do something clever" do
            run_test! do
              expect(foo).to be(true)
            end
          end
        end
      RUBY
    end
  end
```

Failure:
```
Failures:

  1) RuboCop::Cop::RSpec::EmptyExampleGroup when a custom include method takes can a block argument ignores an empty example group with a custom include and a block
     Failure/Error: super
     
       expected: "describe Foo do\n  context \"when I do something clever\" do\n    run_test! do\n      expect(foo).to be(true)\n    end\n  end\nend\n"
            got: "describe Foo do\n  context \"when I do something clever\" do\n  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^...pty example group detected.\n    run_test! do\n      expect(foo).to be(true)\n    end\n  end\nend\n"
     
       (compared using ==)
     
       Diff:
       @@ -1,5 +1,6 @@
        describe Foo do
          context "when I do something clever" do
       +  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Empty example group detected.
            run_test! do
              expect(foo).to be(true)
            end
       
     # ./spec/support/expect_offense.rb:17:in `expect_no_offenses'
     # ./spec/rubocop/cop/rspec/empty_example_group_spec.rb:236:in `block (3 levels) in <top (required)>'
```

**Comment by pirj** [comment]
I guess `run_test!` is an alias or a wrapper around `it`, example description method?

This is going to be resolved with #956 with configurable RSpec DSL. I.e. you'll be able to extend your configuration:
```yaml
AllCops:
  RSpec:
    Language:
      Examples:
        Regular:
        - 'run_test!'
```

**Comment by jtannas** [comment]
> I guess `run_test!` is an alias or a wrapper around `it`, example description method?

yup, it's a wrapper that essentially does:
```ruby
it 'matches the expected format' do
	submit_request
	assert_response_matches_metadata # http code, headers, json structure, etc...
	yield
end
```
	
See here for details: https://github.com/rswag/rswag#paths-operations-and-responses

**Comment by pirj** [comment]
Got it. I recommend disabling the cop temporarily and turning it back on when #956 is merged and released. It's going to happen in 2.0, and I'd love to say that it will happen really soon, but we depend on the parent project with the major release.

**Comment by pirj** [comment]
Duplicated by #1050 

#956 has been released as part of [2.0](https://github.com/rubocop-hq/rubocop-rspec/releases/tag/v2.0.0), `CustomIncludeMethods` have been removed in #1007 that is also part of 2.0.

Please let us know if `RSpec/EmptyExampleGroup` works properly now, and feel free to reopen if it doesn't.
